src/tiledb/cloud/asset.py

```diff
@@ -11,30 +11,30 @@
 from .rest_api.models import GroupInfo  # type: ignore
 
 
-def delete(uri: str, recursive: bool = False) -> None:
+def delete_asset(uri: str, recursive: bool = False) -> None:
     """Deregister the asset and remove its physical groups and arrays from storage.
 
     :param uri: URI of the asset.
     :return: None.
     """
-    delete_map = {
+    delete_map: Mapping[str, Callable] = {
         "array": array.delete_array,
         "group": partial(groups.delete, recursive=recursive),
     }
-    asset_type = tiledb.object_type(uri, ctx=tiledb.cloud.Ctx())
+    asset_type: str = tiledb.object_type(uri, ctx=tiledb.cloud.Ctx())
     func = delete_map[asset_type]
     return func(uri)
 
 
-def info(uri: str) -> Union[ArrayInfo, GroupInfo]:
+def asset_info(uri: str) -> Union[ArrayInfo, GroupInfo]:
```

Comment:
As soon as I realized we want to export this from tiledb.cloud for academy examples it had to be renamed to avoid a clash with `array.info`.

Reply:
What was the clash? Aren't they separated by `tiledb.cloud.asset.info` and `tiledb.cloud.array.info` (and `tiledb.cloud.group.info`)? Or is there export issues?

Reply:
Although this is mostly a detail, I do prefer having the same API for everything and differentiate in the import statements if we are going to use both:
Either:
```python
from tiledb.cloud import asset
from tiledb.cloud import array

asset.info(...)
array.info(...)
```  
Or:
```python
from tiledb.cloud.asset import info as asset_info
from tiledb.cloud.array import info as array_info

asset_info(...)
array_info(...)
```